example/src/App.tsx

```diff
@@ -1093,6 +1094,14 @@ class App extends React.Component<{}, State> {
       { text: 'How are you?', icon: 'alert' },
     ];
 
+    const events: CalendarEvent[] = [
+      {
+        title: 'Birthday',
+        start: new Date(2020, 9, 6),
+        end: new Date(2020, 9, 7),
```

Comment:
@csumudu Are these dates are hard coded?

Reply:
Yes, it's a example page with dummy data just to test the widget